### DISCUSSION TITLE
Swipe config is broken

### DISCUSSION THREAD

**Discussion by NoWorries** [community/community#118355] [Mobile]
Changing links are adjusting the wrong example, video attached.

Changing the left swioe, updates the right swipe. And vice versa.

https://github.com/community/community/assets/803064/a661f2fc-34c3-4c7d-9b2c-4207e7d17104

**Comment by jfullstackdev** [comment]
in my view, I cannot play it, so I can't see the actual problem

![image](https://github.com/community/community/assets/47092464/84d7dff0-0617-4016-98c2-8bb449e8852f)

**Reply by NoWorries** [reply]
Not sure why video not playing for you. Here is a sequence showing me try to change Right Swipe and it changes the Left Swipe visual.

Notice the visual and the selection do not match up, it's put them in wrong spots.

![VideoCapture_20240409-191755.jpg](https://github.com/community/community/assets/803064/fd22cb85-eca2-4281-a82e-c85e70b331c4)

![VideoCapture_20240409-191821.jpg](https://github.com/community/community/assets/803064/d217832b-57dd-495b-af2c-fa2a8483727f)

![VideoCapture_20240409-191824.jpg](https://github.com/community/community/assets/803064/f22f91a6-f1fe-4bf1-bb65-d536066e5bb9)

**Reply by jfullstackdev** [reply]
here is mine, I cannot reproduce it, though I believe it's really a bug

![Screenshot_2024-04-09-16-09-36-80_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/community/community/assets/47092464/55b1615c-cfe3-4f39-9b63-021ec58cc93e)
